code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1373,6 +1373,8 @@
 		ACL |= I.GetAccess()
 	if(wear_id)
 		ACL |= wear_id.GetAccess()
+	if(head && istype(head, /obj/item/clothing/head/bearpelt/real/spare))
```

Comment:
performance-wise it costs us basically the same to check to see if the head item is a sparebear pelt vs doing a GetAccess() on the head item; since there could theoretically be future head-based access it makes more sense to do this is in the more flexible fashion

Reply:
I think I remember the reason I did that was in case other things that could have access could be worn on the head other than just spare bear pelts, but in any case it's now just a head check.